src/main/java/com/ft/methodearticleinternalcomponentsmapper/messaging/MessageProducingInternalComponentsMapper.java

```diff
@@ -37,7 +35,7 @@ void mapInternalComponents(EomFile methodeContent, String transactionId, Date me
             message = messageBuilder.buildMessage(
                     internalComponentsMapper.map(methodeContent, transactionId, messageTimestamp, false)
             );
-        } catch (MethodeArticleMarkedDeletedException | MethodeArticleHasNoInternalComponentsException e) {
+        } catch (MethodeArticleMarkedDeletedException e) {
```

Comment:
Since the MethodeArticleHasNoInternalComponentsException can no longer be thrown we should change the message that is being logged back to "Article {} is marked as deleted.". The name of the messageBuilder method should go back to the previous name as well.